schema.graphql

```diff
@@ -1,313 +1,337 @@
+    id: ID!
+    numConditions: Int!
+    numOpenConditions: Int!
+    numClosedConditions: Int!
+
+    "Number of unique traders interacting with Polymarket"
+    numTraders: BigInt!
+
+    "Number of trades of any kind for all market makers"
+    tradesQuantity: BigInt!
+    "Number of purchases of shares from any market maker"
+    buysQuantity: BigInt!
+    "Number of sales of shares to any market maker"
+    sellsQuantity: BigInt!
+
+    "Global volume in USDC base units"
+    collateralVolume: BigInt!
+    "Global volume in USDC scaled by 10^6"
+    scaledCollateralVolume: BigDecimal!
+    "Global fees in USDC base units"
+    collateralFees: BigInt!
+    "Global fees in USDC scaled by 10^6"
+    scaledCollateralFees: BigDecimal!
+
+    "Global volume of share purchases in USDC base units"
+    collateralBuyVolume: BigInt!
+    "Global volume of share purchases in USDC scaled by 10^6"
+    scaledCollateralBuyVolume: BigDecimal!
+    "Global volume of share sales in USDC base units"
+    collateralSellVolume: BigInt!
+    "Global volume of share sales in USDC scaled by 10^6"
+    scaledCollateralSellVolume: BigDecimal!
 }
 
 type Account @entity {
-  "User address"
-  id: ID!
-  "Timestamp at which account first interacted with Polymarket"
-  creationTimestamp: BigInt!
-  "Timestamp at which account most recently interacted with Polymarket"
-  lastSeenTimestamp: BigInt!
-  "Total volume of this user's trades in USDC base units"
-  collateralVolume: BigInt!
-  "Total number of trades performed by this user"
-  numTrades: BigInt!
-  "Total volume of this user's trades in USDC scaled by 10^6"
-  scaledCollateralVolume: BigDecimal!
-  "Timestamp of last Buy or Sell transaction"
-  lastTradedTimestamp: BigInt!
-  "Markets in which user has provided liquidity"
-  fpmmPoolMemberships: [FpmmPoolMembership!] @derivedFrom(field: "funder")
-  "Markets in which the user has taken a position on the outcome"
-  marketPositions: [MarketPosition!] @derivedFrom(field: "user")
-  "Purchases and sales of shares by the user"
-  transactions: [Transaction!] @derivedFrom(field: "user")
-  "Split of collateral / outcome tokens into multiple positions"
-  splits: [Split!] @derivedFrom(field: "stakeholder")
-  "Merge of more specific outcome tokens into collateral / more general outcome tokens"
-  merges: [Merge!] @derivedFrom(field: "stakeholder")
-  "Redemption of underlying collateral after a market has resolved"
-  redemptions: [Redemption!] @derivedFrom(field: "redeemer")
+    "User address"
+    id: ID!
+    "Timestamp at which account first interacted with Polymarket"
+    creationTimestamp: BigInt!
+    "Timestamp at which account most recently interacted with Polymarket"
+    lastSeenTimestamp: BigInt!
+    "Total volume of this user's trades in USDC base units"
+    collateralVolume: BigInt!
+    "Total number of trades performed by this user"
+    numTrades: BigInt!
+    "Realized profit in USDC from merges, sells, and for positions in resolved markets"
```

Comment:
any idea why this is not syntactically highlighted correctly here? seems to be the comma, I assume it's ok though.

Reply:
@L-Kov Must be that, looks fine in local IDE